load_postgres_data.py

```diff
@@ -0,0 +1,74 @@
+import os
+import sys
+import time
+import subprocess
+import signal
+import tempfile
+import getpass
+
+conf = {
+    'staging': {
```

Comment:
This will not pass flake8. BTW as soon as we put it under `server/athenian/api` you'll notice this :)